bionlp/taggers/rnn_feature/tagger_utils.py

```diff
@@ -123,6 +123,41 @@ def trim_tags(tagged_data):
                           for x, y in tagged_data[i]]
     return tagged_data
 
+def get_regular_or_capitalized_embeddings(w2i, i2w, mdl, params):
+    embedding_vector_size = params['emb1_size']
+    embeddings = []
+    for i in range(len(w2i)):
+        term_emb = None
+        if i in i2w:
```

Comment:
If you'd iterate over the elements of `i2w` in line 129, you can get rid of this check, or am I missing something?